assets/js/modules/adsense/components/setup/v2/SetupAccount.js

```diff
@@ -70,23 +70,29 @@ export default function SetupAccount( { account } ) {
 		select( MODULES_ADSENSE ).getAFCClient( accountID )
 	);
 
-	const afcClientID = determineClientID( { clients: clients || [] } );
+	const { setClientID, setAccountStatus, setSiteStatus } = useDispatch(
+		MODULES_ADSENSE
+	);
 
-	const { setClientID, setAccountStatus } = useDispatch( MODULES_ADSENSE );
+	useEffect( () => {
+		if ( afcClient?._id ) {
```

Comment:
This condition also needs to check that `clientID !== afcClient._id`
```suggestion
		if ( afcClient?._id && clientID !== afcClient._id ) {
```